### PR TITLE
Fix styles outside of paragraphs, introduce "purple bar"

### PR DESCRIPTION
**Pull request by tegud**
Small change to begin with...

### CODE REVIEW

**Comment by azdlowry** [comment]
Think we should add the people bar to the other page types as well. Author and tag.

File: laterooms/page.hbs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+<div class="container" id="content" role="main">
+
+    <article class="{{post_class}}" id="post-{{id}}">
+		{{content}}
+    </article>
+	
+	{{/post}}
+	
+    <div id="disqus_thread"></div>
```
Comment (azdlowry) [inline laterooms/page.hbs]: Do we want the disqus thread on static pages?

**Comment by tegud** [comment]
Eurgh, you and your consistency! It's like there's more than the three random pages I clicked on...

File: laterooms/page.hbs (hunk 1 above)
Comment (tegud) [inline laterooms/page.hbs]: And it is ATM :)

**Pull request merged**